website/docs/advanced/tenancy.md

```diff
@@ -11,37 +11,63 @@ This feature is in alpha. Use at your own risk.
 ::::
 
 ::::info
-This feature is available from 2024.2 and is not to be confused with brands, which used to be called tenants.
+This feature is available from 2024.2 and is not to be confused with [brands](../core/brands.md), which were previously called tenants.
 ::::
 
-## Preparations
+## About tenants
 
-Starting with 2024.2, authentik allows an administrator or operator to create multiple tenants. This means that an operator can manage several authentik installations without having to deploy additional instances.
+Starting with version 2024.2, authentik allows an administrator or operator to create multiple tenants. This means that an operator can manage several different and distinct authentik installations, each with it's own Install ID and license(s). The relationships between tenant and installation (Install ID) is a 1:1 relationship, so for each tenant, there is a unique Install ID.
```

Comment:
So the relationships between an installation, tenants, and licenses are as follows (the leftmost items contain the items under them):
- Authentik installation (i.e. a docker-compose or kubernetes deployment)
  - Install ID (unique to the installation)
  - Default Tenant
    - License 1
    - License 2
    - Brand 1
    - Brand 2
  - Tenant 1
    - [...]
  - Tenant 2
    - [...]

Reply:
Yep, that is the relationship I finally understood yesterday (thanks to your patience), and attempted to say in the words above. Do you see any word changes that need to be made?

Reply:
I don't really know tbh. Whatever you think is best to explain this